libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/TriggerConditions/OnCondition.cs

```diff
@@ -60,6 +68,35 @@ public OnCondition(string condition = null, List<Dialog> actions = null, [Caller
         [JsonIgnore]
         public virtual SourceRange Source => DebugSupport.SourceMap.TryGetValue(this, out var range) ? range : null;
 
+        /// <summary>
+        /// Gets or sets the rule priority expression where 0 is the highest and less than 0 is ignored.
```

Comment:
>0 is the highest [](start = 60, length = 16)

really?  That doesn't seem right..

Reply:
We use priority 0 for our planning and it works nicely with position in a list...

---
In reply to: [360177333](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360177333) [](ancestors = 360177333)